de_workloads/ingest/ingest_azure_sql_example/config/ingest_sources/ingest_config.json

```diff
@@ -43,7 +43,7 @@
             "schema": "movies",
             "table": "ratings_small",
             "columns": "[userId], [movieId], [rating], [timestamp]",
-            "load_type": "delta",
+            "load_type": "full",
```

Comment:
Full or delta in the example?

Reply:
the silver pipeline was failing because it was expecting all the bronze data to be in a "full" folder